env/custom.sh

```diff
@@ -1,2 +1 @@
-# set e.g.
-# export CBIOPORTAL_URL="http://localhost:8080"
+export BACKEND_BRANCH=thehyve:rfc56_pdannotations_for_cna
```

Comment:
make sure you remove this before merging (right?)

Reply:
or should this just be added to .gitignore?

Reply:
`env/custom.sh` is already in .gitignore. The reason for adding this is that the e2e-localdb tests need this to correctly evaluate the backend to use for tests. It will be deleted when switching this PR from draft to final before merge.

Reply:
then there is something wrong with .gitignore because by definition if it's ignored by git it should not be showing up in the pull request, right?